tests/sqllogictests/suites/mode/standalone/explain/lazy_read.test

```diff
@@ -0,0 +1,82 @@
+statement ok
+set lazy_read_threshold=100
+
+statement ok
+drop table if exists t_lazy
+
+statement ok
+create table t_lazy (a int, b float, c string, d tuple(a int, b int), e date)
+
+query T
+explain select * from t_lazy order by a desc limit 2
```

Comment:
What's the explain when `set lazy_read_threshold=0`?

Reply:
There will not be `RowFetch` if `lazy_read_threshold` is set to 0.

Reply:
Can we add the case to the test, it's very useful to know what's the difference from the explain.

Reply:
Ok